src/main/java/org/symphonyoss/integration/webhook/salesforce/parser/BaseSalesforceParser.java

```diff
@@ -263,4 +282,34 @@ private boolean emailExistsAtSymphony(String emailAddress) {
     return user.getId() != null;
   }
 
+  private String getEmailLastModifiedBy(JsonNode node) {
+    return node.path(SalesforceConstants.LAST_MODIFY_BY).path(SalesforceConstants.EMAIL).asText();
+  }
+
+  protected SafeString getEmailLastModifiedByFormatted(JsonNode node) {
+    String emailLastModifiedBy = getEmailLastModifiedBy(node);
+
+    if (!emailExistsAtSymphony(emailLastModifiedBy)) {
+      return EMPTY_SAFE_STRING;
+    }
+
+    return presentationFormat(MessageMLFormatConstants.MESSAGEML_MENTION_EMAIL_FORMAT, emailLastModifiedBy);
+  }
+
+  protected SafeString getFieldsUpdated(JsonNode node) {
```

Comment:
Is there a unit test class for this class?  I see new methods here, but no new unit tests.